users/views.py

```diff
@@ -1,3 +1,72 @@
-from django.shortcuts import render
+import json
+import bcrypt
+import jwt
 
-# Create your views here.
+from django.http              import JsonResponse
+from django.views             import View
+from core.utils               import *
+
+from django.core.exceptions   import ValidationError 
+from wesop.settings           import SECRET_KEY,ALGORITHM
+from core.validator import *
+
+from users.models import *
+
+
+class SignupView(View): 
+    def post(self,request):
+        
+        try:
+            
+            data       = json.loads(request.body)   
+            email      = data['email'] 
+            first_name = data['first_name'] 
+            last_name  = data['last_name']
+            password   = data['password']
+            
+            
+            if User.objects.filter(email = email).exists():
+             return JsonResponse({"message":"DUPLICATE_EMAIL"}, status = 400)
```

Comment:
- 공백 1칸
- indent가 안 맞는 부분이 있습니다. 확인해 주세요.
```suggestion
            password   = data['password']
            
            if User.objects.filter(email = email).exists():
                return JsonResponse({"message":"DUPLICATE_EMAIL"}, status = 400)
```